site/css/my.css

```diff
@@ -92,15 +92,41 @@ header.navbox nav a.search{
     height:33px;
     background:url(../images/search.svg) top left no-repeat
 }
+.leaflet-popup-content-wrapper{
+    background: rgba(255, 255, 255, 0.87) !important;    
+}
 ul{
-    list-style:disc outside none;
+    list-style:none outside none;
+    padding-left: 20px;
+    margin-bottom: 0px;
```

Comment:
This also affects the line after the GitHub link in the information window (the three vertical bars).

Should therefore be added as:
```css
.leaflet-popup-content ul{
     margin-bottom: 0px;
}
```